media-sound/apulse/files/apulse-0.1.13-libgen.patch

```diff
@@ -0,0 +1,13 @@
+Fix build with musl 1.2.5
+
+--- a/src/apulse-misc.c
++++ b/src/apulse-misc.c
+@@ -22,6 +22,8 @@
+  * SOFTWARE.
+  */
+ 
++#include <libgen.h>
```

Comment:
This may fix the build, but it potentially uses the POSIX implementation of basename instead of the GNU implementation. And if the code that uses basename is not prepared to do that, random crashes may occur.
I.e., this is not a valid solution.

Reply:
Code that is using basename:
Gets sting it operates on by readlinking `/proc/self/exe` so it can't be empty or only `/` or has tailing `/`, it copies results of the operation into buffer and throws away original string.
So, it's prepared to deal with POSIX implementation and it's a valid fix.